slider/src/sbs.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable */
+require('./layout.css');
+require('./range.css');
+
+var mapWasDragEnabled;
+var mapWasTapEnabled;
+
+// Leaflet v0.7 backwards compatibility
+function on (el, types, fn, context) {
+  types.split(' ').forEach(function (type) {
+    L.DomEvent.on(el, type, fn, context);
+  });
+}
+
+// Leaflet v0.7 backwards compatibility
+function off (el, types, fn, context) {
+  types.split(' ').forEach(function (type) {
+    L.DomEvent.off(el, type, fn, context);
+  });
+}
+
+function getRangeEvent (rangeInput) {
+  return 'oninput' in rangeInput ? 'input' : 'change';
+}
+
+function cancelMapDrag () {
+  mapWasDragEnabled = this._map.dragging.enabled();
+  mapWasTapEnabled = this._map.tap && this._map.tap.enabled();
+  this._map.dragging.disable();
+  this._map.tap && this._map.tap.disable();
+}
+
+function uncancelMapDrag (e) {
+  this._refocusOnMap(e);
+  if (mapWasDragEnabled) {
+    this._map.dragging.enable();
+  }
+  if (mapWasTapEnabled) {
+    this._map.tap.enable();
+  }
+}
+
+// convert arg to an array - returns empty array if arg is undefined
+function asArray (arg) {
+  return (arg === 'undefined') ? [] : Array.isArray(arg) ? arg : [arg];
+}
+
+function noop () {
+  return;
+}
+
+L.Control.SideBySide = L.Control.extend({
+  options: {
+    thumbSize: 42,
+    padding: 0
+  },
+
+  initialize: function (leftLayers, rightLayers, options) {
+    this.setLeftLayers(leftLayers);
+    this.setRightLayers(rightLayers);
+    L.setOptions(this, options);
+  },
+
+  getPosition: function () {
+    var rangeValue = this._range.value;
+    var offset = (0.5 - rangeValue) * (2 * this.options.padding + this.options.thumbSize);
+    return this._map.getSize().x * rangeValue + offset;
+  },
+
+  setPosition: noop,
+
+  includes: L.Mixin.Events,
```

Comment:
Seeing a deprecation warning to use `L.Evented` instead.

Reply:
I'll see if it breaks anything - I didn't change it because 99% of this is copy pasted from leaflet-side-by-side.